src/framework/audio/iplayer.h

```diff
@@ -50,6 +50,29 @@ class IPlayer
     virtual async::Channel<TrackSequenceId, PlaybackStatus> playbackStatusChanged() const = 0;
 };
 
+using IPlayerHandlerPtr = std::shared_ptr<IPlayerHandler>;
+
+class IPlayer
```

Comment:
This interface is almost an exact copy of ISequencePlayer. Perhaps we should also do something with ISequencePlayer...